src/main/java/org/sonarlint/intellij/server/RequestProcessor.kt

```diff
@@ -49,8 +53,13 @@ class RequestProcessor(private val appInfo: ApplicationInfo = ApplicationInfo.ge
     }
 
     private fun getStatusData(): Response {
-        val description = ProjectManager.getInstance().openProjects.joinToString(", ") { it.name }
-        val status = Status("${appInfo.versionName} ${appInfo.fullVersion}", description)
+        var description = appInfo.fullVersion
+        val edition = ApplicationNamesInfo.getInstance().editionName
+        if (edition != null) {
+            description += " ($edition)"
+        }
+        description += " - " + ProjectManager.getInstance().openProjects.joinToString(", ") { it.name }
```

Comment:
You can have zero projects opened (on the welcome screen).

Reply:
Fixed